_data/jobs/iris-hep-student-fellow.yaml

```diff
@@ -0,0 +1,17 @@
+title: Student Fellow
+text: |
+  Horizontal scaling of HTTP caches: learn how to automate load-balancing with Kubernetes.
+  
+  The OSG offers an integrated software stack and infrastructure used by the High-Energy Physics
+  community to meet their computational needs. Frontier Squid is part of this software stack and
+  acts as an HTTP proxy, caching requests to improve network usage. We seek a fellow to turn our
+  existing single cache Kubernetes deployment into one that can scale horizontally using the same
+  underlying storage for its cache.
+  
+  [Details and Previous Fellows.](https://iris-hep.org/fellows.html)
+button:
+  href: mailto:bbockelman@morgridge.org,blin@cs.wisc.edu,mat@cs.wisc.edu
+  text: Apply by Emailing Mentors
+type: internal
+institution: OSG & IRIS-HEP
```

Comment:
This field should consistently be an actual institution, not projects. So… Morgridge and UW–Madison here, right?